### ISSUE TITLE
I'm assuming you don't need the "Waterfox Limited" folder, is there a problem with deleting it?

### ISSUE THREAD

**Issue by Shitennouji** [MrAlex94/Waterfox#2359]
### What happened?

A bug happened!
I am a Windows user (20H2) and after upgrading to G4.0.0, a new G4.0.0 Profile folder was created in (C:\Users\AppDataRoaming\Waterfox Limited\Waterfox), but When I applied the minor update "G4.0.1", it overwrote and updated the Profile in the previous directory (C:\Users\AppDataRoaming\Waterfox). This action is preferable, however, there is a folder of directory that seem unnecessary.

I'm assuming you don't need the "Waterfox Limited" folder, is there a problem with deleting it?
If there are specific instructions (official statements), it will be a revelation to users who have encountered similar events.

### Reproducible?

- [ ] I have tested this on Firefox.

### Version

G4.𝑋 (Future)

### What platform are you seeing the problem on?

Windows

**Comment by MrAlex94** [comment]
You should be fine deleting that folder, although it may get recreated again by extensions and/or update caches.

**Comment by Shitennouji** [comment]
> You should be fine deleting that folder, although it may get recreated again by extensions and/or update caches.

Thanks MrAlex94 for the reply. 
However, This thread was closed without the consent of the Issue Submitter along with a simple answer, but I am not satisfied with that answer.

I'm aware that a new installation will create a new profile, but this is the first time I've experienced a case where an update or upgrade “replaces” a folder with a new profile or a new folder in a different path.
I'm a long-time regular user of Firefox, and use LibreWolf and Pale Moon, and I don't know of any such cases!

From your answer, is this a normal thing in Waterfox?
If updates and upgrades do not update existing profiles, then “personal data and customizations from before, such as settings, bookmarks, extensions, etc.” will not be inherited, and it is very inconvenient to have to migrate, reconfigure, and re-add them.

My original topic was that I saw this case as an accidental and unexpected incident (inadequate quality control from G3 to G4), but if it will continue to occur normally, then the “topic” will change. I will rephrase it as, I would like to see these low-tech methods of behavior changed.
In short, the argument is that “updates and upgrades should be done with existing profiles”.

Related Articles:
Dedicated profiles per Firefox installation | Firefox Help
https://support.mozilla.org/en-US/kb/dedicated-profiles-firefox-installation#firefox:win10:fx91
Back up and restore information in Firefox profiles | Firefox Help
https://support.mozilla.org/en-US/kb/back-and-restore-information-firefox-profiles
